### ISSUE TITLE
Invalid purls and missing titles 

### ISSUE THREAD

**Issue by simonjupp** [OBOFoundry/OBOFoundry.github.io#95]
The following don't have purls that don't resolve. I also noticed a couple have title missing, I'm currently assuming that every ontology should have a title, is this not the case?

- CranioMaxilloFacial http://purl.obolibrary.org/obo/cmf.owl doesn't resolve 
- http://purl.obolibrary.org/obo/mop.owl no title and doesn't resolve 
- No product defined for NCI Thesaurus
- Medaka Developmental Stages http://purl.obolibrary.org/obo/olatdv.owl doesn't resolve 
- http://www.berkeleybop.org/ontologies/ovao.owl no title and doesn't resolve

**Comment by cmungall** [comment]
@mellybelly is cmf active?

**Comment by mellybelly** [comment]
I think cmf is obsolete, and development is now in OCDM, will email Todd Detwiler (don't think he's on github?)

**Comment by markengelstad** [comment]
I'd like to continue to reserve the namespace for CMF Ontology, but at this time, its not clear exactly what domain activity it will have, and it's not currently being developed. 
Is that possible?

**Comment by mellybelly** [comment]
(with apologies I thought for some reason that the cmf namespace was the predecessor to OCDM)

**Comment by cmungall** [comment]
cmf issue resolved: we now have an entry 'reserved' for it (own md file), but it has no product listed. The ontologies.yaml file has been fixed, and now omits any ontology_purl entry for an ontology with no products. @simonjupp uses this field for OLS so we should be good.

For ncit, we have no product listed. Since we already have the ontology grandfathered in, shall we add a product URL (non-OBO)? Does OLS want to include this Simon?

**Comment by cmungall** [comment]
I think most of these should be resolved when the next central build runs at midnight. Once #96 is done this sort of thing should be less common

**Comment by cmungall** [comment]
>    CranioMaxilloFacial http://purl.obolibrary.org/obo/cmf.owl doesn't resolve
removed product
(ie ontology reserved but there is no owl out there)

>    http://purl.obolibrary.org/obo/mop.owl no title and doesn't resolve
resolves

>    No product defined for NCI Thesaurus
deliberate for now. Open a new ticket to coordinate adding to OLS

>    Medaka Developmental Stages http://purl.obolibrary.org/obo/olatdv.owl doesn't resolve

fixed


    http://www.berkeleybop.org/ontologies/ovao.owl no title and doesn't resolve
